### PR TITLE
Update pf4 wizard pf3 tabs

### PR DESCRIPTION
**Pull request by Hyperkid123**
### Changes
- removed demo publish condition
  - we will publish everything to demo site 
- addressed visual feedback on PF4 wizard
  - for wizard buttons customization added option to specify custom CSS class
  - added option to customize controls text
  - next button is disabled until current step is valid
  - adjusted sizes of default Wizard and Step titles
- Added visual feedback for invalid tab content for PF3 tabs
  - user can add a list of validated fields
  - no simple way to handle this by default (schema is too flexible)
screenshot-localhost-8080-2019 04 24-17-29-10

### CODE REVIEW

File: packages/react-renderer-demo/src/editor-demo/common/examples-texts/tabs.js [hunk 1]
```diff
@@ -18,4 +18,17 @@ export default ({ activeMapper }) =>
     </Typography>
     <pre>TAB_ITEM</pre> <Typography variant="body1">as an import from componentTypes</Typography>
     <pre>tab-item</pre> <Typography variant="body1">as a string</Typography>
+    { activeMapper === 'pf3' &&
+    (
+      <Fragment>
+        <Typography variant="h6" gutterBottom>
+        Validation
+        </Typography>
+        <Typography variant="body1">Because of shcema flexibility there is no simple and efficient way to signal invalid tab content.</Typography>
```
Comment (martinpovolny) [inline packages/react-renderer-demo/src/editor-demo/common/examples-texts/tabs.js]: shcema --> schema

File: packages/pf4-component-mapper/src/form-fields/wizard/wizard.js [hunk 2]
```diff
@@ -29,7 +31,8 @@ class Wizard extends React.Component {
   findCurrentStep = activeStep => this.props.fields.find(({ stepKey }) => stepKey === activeStep)
 
   render() {
-    const { title, description, FieldProvider, formOptions } = this.props;
+    const { title, description, FieldProvider, formOptions, buttonLabels, buttonsClassName } = this.props;
+    console.log('Foo: ', buttonsClassName)
```
Comment (martinpovolny) [inline packages/pf4-component-mapper/src/form-fields/wizard/wizard.js]: forgotton console.log here

**Comment by rvsia** [comment]
Tested PF4 wizard, looks great! However, here is something, which is not related to this PR: I noticed PF4 forms could be submitted by pressing enter in focused fields. Other mappers don't do it. :cry: 

I will look at the tabs now.

**Comment by rvsia** [comment]
Tabs work well! :+1: 

Only one small question: should the error icon be visible immediately or after the validation? Because I can see an error icon but no errors.

image

But if this a targeted behavior, it is ready to merge, good job!

File: packages/pf3-component-mapper/src/form-fields/tabs.js [hunk 3]
```diff
@@ -1,18 +1,26 @@
...
+  const errors = formOptions.getState().errors;
+  const hasError = validateFields.find(name => !!errors[name]);
+  return (
+    <NavItem key={ key } eventKey={ index }>
+      { title }
+      { hasError && <Icon style={{ marginLeft: 8, color: '#CC0000' }} type="fa" name="exclamation-circle" /> }
+    </NavItem>
   );
+});
+const renderTabContet = (items, formOptions) =>
```
Comment (rvsia) [inline packages/pf3-component-mapper/src/form-fields/tabs.js]: Contet -> Content

**Comment by Hyperkid123** [comment]
> Tabs work well! 
> 
> Only one small question: should the error icon be visible immediately or after the validation? Because I can see an error icon but no errors.
> 
> But if this a targeted behavior, it is ready to merge, good job!

It should be visible immediately but i think should be on the left! Let me fix that first

**Comment by Hyperkid123** [comment]
@rvsia ok should be fine now

**Comment by rvsia** [comment]
@Hyperkid123 Snapshots failed

**Pull request merged**